src/common/SPELLS/warlock.ts

```diff
@@ -573,6 +573,18 @@ const spells = {
     icon: 'inv-felhound3-shadow-mount',
   },
 
+  // Doom spell (passive debuff applied by Demonbolt when consuming Demonic Core)
+  DOOM: {
```

Comment:
Could rename this to make it more clear that it's the debuff id.
```suggestion
  DOOM_DEBUFF: {
```